region/maridia/outer/Mama Turtle Room.json

```diff
@@ -886,6 +933,52 @@
       ],
       "flashSuitChecked": true,
       "note": "To save energy, start from the upper right ledge; it may require repositioning Mama Turtle to the right to get to the ledge."
+    },
+    {
+      "link": [5, 6],
+      "name": "Base",
+      "requires": [
+        "canUseGrapple",
+        "SpaceJump"
+      ]
+    },
+    {
+      "link": [6, 3],
+      "name": "Base",
+      "requires": []
```

Comment:
Revealing the item and collecting it in one action while falling isn't free.  You can downward shot while falling, but I'd still count that as a hero shot here.
It seems like it would have most of the requirements of "Collect right item with walljump escape".

Reply:
👍 changing to:
```json
    {
      "link": [6, 3],
      "name": "Base",
      "requires": [
        {"or": [
          "canHeroShot",
          "h_canUsePowerBombs",
          "Spazer",
          "Wave"
        ]}
      ]
    },
```